background/src/wallet/wallet.js

```diff
@@ -101,9 +103,9 @@ export default class Wallet {
             },
         })).toString('hex');
 
-        let account = createAccount(wallet.privateKey);
+        let account = createAccount(wallet.privateKey.slice(2));
```

Comment:
```suggestion
        let account = createAccount(wallet.privateKey.replace('0x', ''));
```

I guess it may be more expressive.